requirements.txt

```diff
@@ -1,2 +1,3 @@
 numpy
 torch
+allennlp
```

Comment:
Put allennlp in an extrarequirements.txt file instead along with pandas and tensorboard.

Reply:
`extra_requirements` or `extrasrequirements` ?